bridge/shared/interfaces/secret.ts

```diff
@@ -7,7 +7,7 @@ export interface SecretKeyValuePair {
 
 export interface Secret {
   name: string;
-  scope: SecretScope;
+  scope: SecretScope | string;
```

Comment:
I wonder why this is needed, can you explain?

Reply:
Autocompletion in the IDE. When assigning a value you get the available default ones and can also provide everything else.
Useful for static assignments like Webhook secrets or tests

Reply:
`SecretScope` provides default values which makes auto completion and testing easier, now I see :) :+1: